### ISSUE TITLE
position fixed header blur on window zoom in/out

### ISSUE THREAD

**Issue by Arvn41** [mkoryak/floatThead#421]
In my Table am doing fixed head using following code.

responsiveContainer: function ($table) {
            return $table.closest('.table-responsive');
        }

its work on all scenarios but if any user zoom in or zoom out window screen, headers get blur.

can you please give me any idea so I can fix it.

**Comment by Arvn41** [comment]
Thanks for quick response I have check issues/418 for the same, but you have mention that this is need old version jQuery but we have use latest in our code and its needed,.

Can you give me just idea is it will affect code if I use will-change:transform to will-change:auto; its will solve my issue.

i have use following code after floatThead code.

/$('.floatThead-container').css('willChange', 'auto');

and Thank you very much for great plugin

**Comment by mkoryak** [comment]
sorry, I do not understand this: "Can you give me just idea is it will affect code if I use will-change:transform to will-change:auto; its will solve my issue."

can you rephrase?

**Comment by mkoryak** [comment]
see #418 - wontfix

**Comment by mkoryak** [comment]
No response, closing
